docker/docker-compose.yml

```diff
@@ -116,6 +115,36 @@ services:
       WAIT_HOSTS: solr1:8983
       WAIT_HOSTS_TIMEOUT: 60
 
+
+  mention-storer:
+    build:
+      context: ../mention-storer/
+      dockerfile: dockerfile
+    hostname: mention-storer
+    restart: always
+    environment:
+      WAIT_HOSTS: solr:8983
+
+  mentions-generator:
+    build:
```

Comment:
What is the exact difference between this three line build, and just `build: ../mentions-generator/`?

Reply:
I think in this case there is no difference. `Context` is just the relative path to the directory and the `dockerfile:` is the name of the dockerfile. Since mine are all named dockerfile it doesn't make a difference here.

Reply:
That's what I was getting at, probably redundant and can be replaced by a one-liner.

Reply:
I replaced it. Even tho I thought the readability was better before.

Reply:
I won't force you to change it. But if one knows that the docker file default naming is `Dockerfile`, and that `build` will pick a Dockerfile and build it, I think it's obvious where the path points to ...